client/components/Layout/Images/BlurhashImage.vue

```diff
@@ -23,17 +23,11 @@
         />
       </v-fade-transition>
     </div>
-    <div v-if="!$slots.placeholder" class="absolute icon img">
-      <v-icon
-        class="absolute text--disabled"
-        :size="iconSize"
-        color="white"
-        dark
-      >
+    <slot v-else name="placeholder">
+      <v-icon class="absolute text--disabled" :size="iconSize">
         {{ getItemIcon(item) }}
       </v-icon>
-    </div>
-    <slot v-else name="placeholder" />
+    </slot>
```

Comment:
What if somewhere we don't want an icon placeholder but something else instead?

Reply:
You just use the slot and set whatever you want inside of it.

This is how slots work. What is inside the slot definition is the default. Then you override it by defining a template for that slot when calling the component.